docs/reference/resolver-internals.md

```diff
@@ -25,17 +25,19 @@ there are no general, fast algorithms. In practice, this is misleading for a num
 - When resolution fails, more information is needed than a message that there is no solution (as is
   seen in SAT solvers). Instead, the resolver should produce an understandable error trace that
   states which packages are involved in away to allows a user to remove the conflict.
+- For performance and whether users consider a solution good, the most important heuristic is
```

Comment:
Perhaps rephrase as "The most important heuristic for performance and user experience is..."